### PR TITLE
fixes starting space issue before "Looking to fund"

### LINKED ISSUE #5059: fixing failed checks for components "Translations & Javascript" on weblate
i would like to fix issues of "Failed check: Starting spaces; Trailing question, space, exclamation, colon"

[source1](https://hosted.weblate.org/projects/gitcoin/translations/hi/) [source2](https://hosted.weblate.org/projects/gitcoin/javascript/hi/)

if it is OK to fix them
way 1
create issue for each failed check & pull request

way 2
pull request with commit message, for example:
removed trailing space at assets/onepager/js/receive.js:5

full stop at end of hindi language sentences are not   norm. should i remove or replace "vertical line" with trailing stop if source contains at end of sentence.

### PR DESCRIPTION
**Pull request by leela52452**
##### Description
removed space before "Looking to fund"

##### Refers/Fixes

#5059

##### Testing

i hope it is not necessary

### CODE REVIEW

File: app/dashboard/templates/ens/ens_rate_limit.html [hunk 1]
```diff
@@ -10,7 +10,7 @@ <h2><label>{% trans "Address: "%}{{ owner }}</label></h2>
   </div>
   <div>
     <br>
-    {% trans "Only 1 request is allowed every "%}{{ limit_reset_days }}{% trans " days. If you have lost access to this wallet,"%} 
+    {% trans "Only 1 request is allowed every "%}{{ limit_reset_days }}{% trans "days. If you have lost access to this wallet,"%}
```
Comment (thelostone-mc) [inline app/dashboard/templates/ens/ens_rate_limit.html]: I believe this space is needed here!

Comment (leela52452) [reply app/dashboard/templates/ens/ens_rate_limit.html]: i have edited file solely on the comments in weblate. i do not know coding. due to limit_reset_days a space is needed before the text, i guess. please excuse.

Comment (leela52452) [reply app/dashboard/templates/ens/ens_rate_limit.html]: reverted changes with commit cfc87a82e58657ef22a208c2a3e7f31c3e2f7e76

**Comment by thelostone-mc** [comment]
@leela52452 fixed up the mistakes ! this looks solid now ^_^
thanks

**Pull request merged**